lib/api/src/modules/url.ts

```diff
@@ -42,51 +35,69 @@ let prevParams: ReturnType<typeof queryFromLocation>;
 const initialUrlSupport = ({
   state: { location, path, viewMode, storyId: storyIdFromUrl },
 }: ModuleArgs) => {
-  const addition: Additions = {};
+  const layout: Partial<Layout> = {};
+  const ui: Partial<UI> = {};
   const query = queryFromLocation(location);
   let selectedPanel;
 
   const {
     full,
     panel,
     nav,
-    addons,
-    panelRight,
-    stories,
+    shortcuts,
     addonPanel,
+    addons, // deprecated
+    panelRight, // deprecated
+    stories, // deprecated
     selectedKind,
     selectedStory,
     path: queryPath,
     ...otherParams
   } = query;
 
-  if (full === '1') {
-    addition.isFullscreen = true;
+  if (full === 'true' || full === '1') {
+    layout.isFullscreen = true;
   }
   if (panel) {
     if (['right', 'bottom'].includes(panel)) {
-      addition.panelPosition = panel;
-    } else if (panel === '0') {
-      addition.showPanel = false;
+      layout.panelPosition = panel;
+    } else if (panel === 'false' || panel === '0') {
+      layout.showPanel = false;
     }
   }
-  if (nav === '0') {
-    addition.showNav = false;
+  if (nav === 'false' || nav === '0') {
+    layout.showNav = false;
+  }
+  if (shortcuts === 'false' || shortcuts === '0') {
+    ui.enableShortcuts = false;
+  }
+  if (addonPanel) {
+    selectedPanel = addonPanel;
   }
 
-  // Legacy URLs
+  // @deprecated Superceded by `panel=false`, to be removed in 7.0
```

Comment:
Can you add this to MIGRATION.md as well?

Reply:
Done